### DISCUSSION TITLE
Treat Zoomed in Bullets as New List

### DISCUSSION THREAD

**Discussion by Ivcota** [vslinko/obsidian-zoom#17] [Archive]
I don't know how they do it, but in Logseq during the zoom-in, the indent depth is reset so it looks like a new ul.

I'm not sure if this belongs in the outliner discussions or the zoom discussions, but if there was a basically treat the zoomed-in bullets as starting ones, that would be insane.

Then again, maybe this is an idea for when Obsidian releases the WYSIWYG.

_Logseq_
![image](https://user-images.githubusercontent.com/18635074/122650840-e140a680-d0e9-11eb-8643-a801e7a44ee4.png)\

_Obsidian_
![image](https://user-images.githubusercontent.com/18635074/122650896-35e42180-d0ea-11eb-8521-b9944801a8c3.png)

**Comment by vslinko** [comment]
Duplicate of https://github.com/vslinko/obsidian-zoom/discussions/3
